chain-signatures/node/src/protocol/signature.rs

```diff
@@ -413,27 +413,30 @@ impl SignatureManager {
                     Err(err) => {
                         remove.push(sign_id.clone());
 
-                        if generator.request.proposer == self.me {
-                            if generator.request.indexed.timestamp.elapsed()
-                                < generator.timeout_total
-                            {
-                                failed.push(sign_id.clone());
-                                tracing::warn!(?err, "signature failed to be produced; pushing request back into failed queue");
+                        if generator.request.indexed.timestamp.elapsed() < generator.timeout_total {
+                            failed.push(sign_id.clone());
```

Comment:
actually, we should move failed to right after remove. It's weird that it was only on timeout_total. Timeout total is for all retries which isn't correct

Reply:
ahh, yeah for some reason I was reading it as `>= generator.timeout_total` which means it only is failed on all attempts are done